packages/scenes/src/variables/adhoc/AdHocFiltersVariable.tsx

```diff
@@ -17,13 +17,14 @@ import { getEnrichedFiltersRequest } from '../getEnrichedFiltersRequest';
 import { AdHocFiltersComboboxRenderer } from './AdHocFiltersCombobox/AdHocFiltersComboboxRenderer';
 import { wrapInSafeSerializableSceneObject } from '../../utils/wrapInSafeSerializableSceneObject';
 
-export interface AdHocFilterWithLabels extends AdHocVariableFilter {
+export interface AdHocFilterWithLabels<M = any> extends AdHocVariableFilter {
```

Comment:
```suggestion
export interface AdHocFilterWithLabels<M extends Record<string, any> = {}> extends AdHocVariableFilter {
```

It might be helpful to expect an object here